org.aegisub.Aegisub.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: org.aegisub.Aegisub
+runtime: org.gnome.Platform
+runtime-version: '3.32'
+sdk: org.gnome.Sdk
+command: aegisub
+rename-desktop-file: aegisub.desktop
+rename-icon: aegisub
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  # temporarily disabled
+  # wxGLCanvas doesn't work with Wayland yet
+  # https://trac.wxwidgets.org/ticket/17702
+  #- --socket=wayland
+  # Audio access
+  - --socket=pulseaudio
+  # Filesystem access
+  - --filesystem=host
+  # Dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
```

Comment:
I don't think this is needed anymore with https://blogs.gnome.org/mclasen/2019/07/12/settings-in-a-sandbox-world/?

Reply:
TingPing [said](https://blogs.gnome.org/mclasen/2019/07/12/settings-in-a-sandbox-world/#comment-2054):
> Only fdo 19.08 does it atm

However, GNOME 3.32 runtime is based on Freedesktop 18.08.

Reply:
Oh alright, thanks for the info.